spec/ttml1.xml

```diff
@@ -8581,6 +8581,13 @@ agent with another agent that portrays the character.</p>
 <p>The <att>agent</att> attribute of a <el>ttm:actor</el> element must
 reference a <emph>significant</emph> <el>ttm:agent</el> element that denotes the agent
 acting the part of a character.</p>
+<note role="clarification">
+<p>The semantics associated with an <el>ttm:actor</el> element referencing its parent 
+<el>ttm:agent</el> element are not defined. In the case of an actor playing themselves, 
+like Steve Coogan in The Trip playing Steve Coogan, two <el>ttm:agent</el> elements are used instead: 
```

Comment:
The "are used instead" language seems strange - I think this is describing good practice, so this clause could be rephrased:

_the expected practice is to use two `ttm:agent` elements instead:_